test/node/rand/index.ts

```diff
@@ -0,0 +1,25 @@
+import { beta } from '../../../src/rand';
+import { expect } from 'chai';
+import * as tf from '@tensorflow/tfjs-core';
+import '@tensorflow/tfjs-backend-cpu';
+import 'mocha';
+
+describe('Beta Random test',
+  () => { 
+  it('should return tensor wiht one value', () => {
```

Comment:
A little typo here:` it('should return tensor wiht one value')...` should be `it('should return tensor with one value'`